kafka/kafka-users.yaml

```diff
@@ -306,4 +306,37 @@ spec:
           name: device-telemetry
           patternType: literal
         operation: Describe
+        host: "*"
+---
+apiVersion: kafka.strimzi.io/v1beta1
```

Comment:
Should we mention this user in https://github.com/ppatierno/rhte-2019/blob/master/modules/01-kafka-on-openshift.md#users-creation ?

Reply:
Yes you are right I will add it later today ;)